src/core/lib/security/credentials/ssl/ssl_credentials.cc

```diff
@@ -66,12 +68,40 @@ grpc_ssl_credentials::create_security_connector(
   absl::optional<std::string> overridden_target_name =
       args->GetOwnedString(GRPC_SSL_TARGET_NAME_OVERRIDE_ARG);
   auto* ssl_session_cache = args->GetObject<tsi::SslSessionLRUCache>();
+  tsi_ssl_session_cache* tsi_ssl_session_cache =
```

Comment:
micro-nit: Please make the variable name different than the type name, e.g. maybe `session_cache`.

Reply:
Friendly ping. :)